frontend/packages/client/src/pages/Home.js

```diff
@@ -63,7 +63,7 @@ export default function HomePage() {
   });
 
   return (
-    <section className={classNames}>
+    <>
```

Comment:
This is making the tooltip to be next to the navbar: 
<img width="1067" alt="Screen Shot 2022-09-09 at 11 31 24" src="https://user-images.githubusercontent.com/7526740/189374380-3670d3b1-98dd-4cd8-b443-c251b85de5a6.png">

Maybe adding some margin within the TooltipMessage component would work? This component is only to be shown on safari

Reply:
ended up removing `message-container` from `TooltipMessage` since it was only controlling margin bottom. Now it takes a `className` so we just give it `my-6` for even top/bottom margin